frontend/src/components/Notification.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+// import DateTimePicker from 'react-datetime-picker';
+import { ListGroupItem } from 'reactstrap'
+import '../styles/notification.css'
+import { connect } from 'react-redux'
+
+const mapStateToProps = state => ({
+  isPM: state.user.isPM
+})
+
+class Notification extends Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    const { isPM } = this.props
+    return (
+      <ListGroupItem>
+        <p className="person-name">
+          {' '}
+          <b> {this.props.name} </b>{' '}
```

Comment:
I can't think of a better way to do this right now, but I feel like there might be a cleaner way to include spaces here. We'll probably need to look at it later though.